deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py

```diff
@@ -0,0 +1,339 @@
+import argparse
+import os
+import sys
+import pandas as pd
+from prettytable import PrettyTable
+from scipy.stats import ttest_ind, levene, f_oneway
+
+STATS_MAPPING = {
+    'cpu': 'CPU(%)',
+    'memory': 'RSS(KB)',
+    'virtual_memory': 'VMS(KB)',
+    'file_descriptor': 'FD',
+    'read_ops': 'Read_Ops',
+    'write_ops': 'Write_Ops',
+    'disk_read': 'Disk_Read(B)',
+    'disk_written': 'Disk_Written(B)',    
+    'disk_usage': 'Disk(%)',
+    'uss': 'USS(KB)',
+    'pss': 'PSS(KB)',
+    'swap': 'SWAP(KB)',
+}
+
+STATISTICS_LIST = ['Mean', 'Median', 'Max value', 'Min value', 'Standard deviation', 'Variance']
+
+def get_parameters():
+    """Get and process script parameters.
+    
+    Returns:
+        argparse.Namespace: script parameters.
+    """
+    arg_parser = argparse.ArgumentParser()
+
+    arg_parser.add_argument('-b', '--baseline', metavar='<file>', type=str, action='store',
+                            help='Baseline file to compare', required=True, dest='baseline_data')
+
+    arg_parser.add_argument('-f', '--file', metavar='<file>', type=str, action='store',
+                            help='Data file to compare', required=True, dest='data_source')
+    
+    return arg_parser.parse_args()
```

Comment:
This library is not planned to be used as a script. I suggest removing all parsing parameters logic. This should be handled by a class method.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5622/commits/26dc5cfa8d7f2339d13d5d5820d1ad7cceda3625